public/app/listeners/saveAlgorithm.js

```diff
@@ -22,13 +22,6 @@ const saveAlgorithm = (mainWindow, db) => async (event, { metadata, code }) => {
       type: SAVE_ALGORITHM_SUCCESS,
     });
   } catch (err) {
-    if (err.code === 'ENOENT') {
-      logger.error('File not found!');
-      return mainWindow.webContents.send(SAVE_ALGORITHM_CHANNEL, {
-        type: SAVE_ALGORITHM_ERROR,
-        error: ERROR_MISSING_FILE,
-      });
-    }
```

Comment:
Why remove this part?

Reply:
I believe there will never be a missing file error, `fs.writeFileSync` writes into the target location regardless of whether it exists or not.